src/UI/AnvilWindow.cpp

```diff
@@ -49,11 +47,9 @@ void cAnvilWindow::SetRepairedItemName(const AString & a_Name, cPlayer * a_Playe
 
 
 
-void cAnvilWindow::GetBlockPos(int & a_PosX, int & a_PosY, int & a_PosZ)
+void cAnvilWindow::GetBlockPos(Vector3i & a_Pos)
```

Comment:
Ideally, this function should now return the `Vector3i`
```suggestion
const Vector3i & cAnvilWindow::GetBlockPos()
```

Reply:
Could be even moved into the header file now.